test/kubemark/start-kubemark-master.sh

```diff
@@ -16,9 +16,219 @@
 
 # TODO: figure out how to get etcd tag from some real configuration and put it here.
 
-function write_supervisor_conf() {
-	local name=$1
-	local exec_command=$2
+function create-dirs {
+	echo "Creating required directories"
+	mkdir -p /var/lib/kubelet
+	mkdir -p /etc/kubernetes/manifests
+}
+
+# Setup working directory for kubelet.
+function setup-kubelet-dir {
+	echo "Making /var/lib/kubelet executable for kubelet"
+	mount -B /var/lib/kubelet /var/lib/kubelet/
+	mount -B -o remount,exec,suid,dev /var/lib/kubelet
+}
+
+# Formats the given device ($1) if needed and mounts it at given mount point
+# ($2).
+function safe-format-and-mount() {
+	device=$1
+	mountpoint=$2
+
+	# Format only if the disk is not already formatted.
+	if ! tune2fs -l "${device}" ; then
+		echo "Formatting '${device}'"
+		mkfs.ext4 -F -E lazy_itable_init=0,lazy_journal_init=0,discard "${device}"
+	fi
+
+	mkdir -p "${mountpoint}"
+	echo "Mounting '${device}' at '${mountpoint}'"
+	mount -o discard,defaults "${device}" "${mountpoint}"
+}
+
+# Finds the master PD device; returns it in MASTER_PD_DEVICE
+function find-master-pd {
+	MASTER_PD_DEVICE=""
+	if [[ ! -e /dev/disk/by-id/google-master-pd ]]; then
+		return
+	fi
+	device_info=$(ls -l /dev/disk/by-id/google-master-pd)
+	relative_path=${device_info##* }
+	MASTER_PD_DEVICE="/dev/disk/by-id/${relative_path}"
+}
+
+# Mounts a persistent disk (formatting if needed) to store the persistent data
+# on the master -- etcd's data, a few settings, and security certs/keys/tokens.
+# safe-format-and-mount only formats an unformatted disk, and mkdir -p will
+# leave a directory be if it already exists.
+function mount-master-pd {
```

Comment:
we had an option to put event etcd on a separate disk - was it removed, or it's still somewhere?